C#/forSpbu/Matrix-mult/MatrixMultiplier.cs

```diff
@@ -0,0 +1,73 @@
+namespace Matrix_mult;
+
+public static class MatrixMultiplier
+{
+    /// <summary>
+    /// Performs matrix multiplication in single thread
+    /// </summary>
+    /// <param name="first">First matrix in multiplication</param>
+    /// <param name="second">Second matrix in multiplication</param>
+    /// <returns>Result matrix</returns>
+    /// <exception cref="MatrixMultiplierException">If matrices are incompatible(width of first and height of second are not equal)</exception>
+    public static Matrix SingleThreaded(Matrix first, Matrix second)
```

Comment:
Лучше с глаголом назвать
```suggestion
    public static Matrix SingleThreadedMultiply(Matrix first, Matrix second)
```